network-api/networkapi/mozfest/migrations/0039_datawrapper_block.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.1.11 on 2022-02-08 15:40
+
+from django.db import migrations
+import wagtail.core.blocks
+import wagtail.core.blocks.static_block
+import wagtail.core.fields
+import wagtail.documents.blocks
+import wagtail.embeds.blocks
+import wagtail.images.blocks
+import wagtail.snippets.blocks
+import wagtailmedia.blocks
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mozfest', '0038_auto_20220126_2241'),
```

Comment:
file becomes `0040_datawrapper_block.py` and the dependency becomes `0039_auto_20220208_0037`